Packs/ServiceNow/Integrations/ServiceNow_IAM/ServiceNow_IAM.py

```diff
@@ -137,10 +138,12 @@ def get_mapping_fields_command(client):
     return GetMappingFieldsResponse([incident_type_scheme])
 
 
-def get_user_command(client, args, mapper_in):
-    user_profile = IAMUserProfile(user_profile=args.get('user-profile'))
+def get_user_command(client, args, mapper_in, mapper_out):
+    user_profile = IAMUserProfile(user_profile=args.get('user-profile'), mapper=mapper_out)
     try:
-        service_now_user = client.get_user(user_profile.get_attribute('email'))
+        iam_attr, iam_attr_value = user_profile.get_first_available_iam_user_attr(IAM_GET_USER_ATTRIBUTES)
+        service_now_filter_name: str = 'sys_id' if iam_attr == 'id' else iam_attr
```

Comment:
will it return the id if no email/username is provided?

Reply:
Not sure I understand what you meant?

Reply:
if I do `user-profile={\"id\": \"USERID\"}` (USERID is an existing user's ID in Slack) will it work?
Asking because there is no id mapping in the outgoing mapper.